packages/cardano-services/src/Utxo/DbSyncUtxoProvider/mappers.ts

```diff
@@ -0,0 +1,52 @@
+import { Cardano } from '@cardano-sdk/core';
+import { UtxoModel } from './types';
+
+const generateAssetId = (policy: string, name: string) =>
+  Cardano.AssetId(policy + (!name || name === '' ? '\\x' : name));
+
+/**
+ * Indexes the given utxos by its coin id in order to generate core utxo objects
+ *
+ * @param {UtxoModel[]} utxos  obtained utxos rows
+ * @returns {Cardano.Utxo[]} an array of core utxo objects
+ */
+export const toUtxoByAddressesResponse = (utxos: UtxoModel[]): Cardano.Utxo[] => {
```

Comment:
```suggestion
export const utxosToCore = (utxos: UtxoModel[]): Cardano.Utxo[] => {
```

The function just takes in an array of DB data, and returns core types.